workflow/sample_files/base-battery-scheduled.xml

```diff
@@ -445,7 +445,7 @@
               <Units>kWh</Units>
               <Value>18.0</Value>
             </UsableCapacity>
-            <RatedPowerOutput>15000.0</RatedPowerOutput>
+            <RatedPowerOutput>6000.0</RatedPowerOutput>
```

Comment:
All of these changes are going to be overwritten the next time tasks.rb is run. @joseph-robertson, can you update the script?

Reply:
@joseph-robertson I think the battery tests need to be updated too.